shell/platform/android/external_view_embedder/external_view_embedder.cc

```diff
@@ -63,10 +63,10 @@ SkRect AndroidExternalViewEmbedder::GetViewRect(int view_id) const {
   const EmbeddedViewParams& params = view_params_.at(view_id);
   // TODO(egarciad): The rect should be computed from the mutator stack.
   // https://github.com/flutter/flutter/issues/59821
-  return SkRect::MakeXYWH(params.finalBoundingRect().x(),                     //
-                          params.finalBoundingRect().y(),                     //
-                          params.sizePoints().width() * device_pixel_ratio_,  //
-                          params.sizePoints().height() * device_pixel_ratio_  //
+  return SkRect::MakeXYWH(params.finalBoundingRect().x(),      //
+                          params.finalBoundingRect().y(),      //
+                          params.finalBoundingRect().width(),  //
```

Comment:
nice 👍

Reply:
any chance we can just return `params.finalBoundingRect()` from this method?

Reply:
done